README.md

```diff
@@ -466,6 +468,25 @@ block-beta
 ```bash
 protoflow execute ReadFile path=/tmp/file.txt
 ```
+#### [`ReadSocket`]
+
+A block that reads proto messages from a TCP port.
+
+```mermaid
+block-beta
+    columns 4
+    ReadSocket space:2 Sink
+    ReadSocket-- "output" -->Sink
+
+    classDef block height:48px,padding:8px;
+    classDef hidden visibility:none;
+    class ReadSocket block
+    class Sink hidden
+```
+
+```bash
+protoflow execute ReadSocket host="127.0.0.1" port=7077 buffer_size=1024
```

Comment:
The quotes around the host value are unnecessary here

Reply:
fixed